Remove unused legacy GUI shader lookups; getProgram loads the installed shader pair.

--- a/components/myguiplatform/myguirendermanager.cpp
+++ b/components/myguiplatform/myguirendermanager.cpp
@@ -535,12 +535,7 @@
     {
         auto program = shaderManager.getProgram("gui");
 //## VR_PATCH BEGIN
-// VR-TODO: Is there a specific reason i am specifying GLSLVersion here, isn't that taken care of in the shaderManager.getProgram?
-        auto vertexShader
-            = shaderManager.getShader("gui_vertex.glsl", { { "GLSLVersion", "120" } }, osg::Shader::VERTEX);
-        auto fragmentShader
-            = shaderManager.getShader("gui_fragment.glsl", { { "GLSLVersion", "120" } }, osg::Shader::FRAGMENT);
-
+// getProgram loads the installed gui.vert/gui.frag pair.
         mGuiStateSet->setAttributeAndModes(program, osg::StateAttribute::ON);
         mGuiStateSet->addUniform(new osg::Uniform("diffuseMap", 0));
 //## VR_PATCH END
